packages/proxy/lib/http/request-middleware.ts

```diff
@@ -23,35 +23,75 @@ const LogRequest: RequestMiddleware = function () {
   this.next()
 }
 
+const CorrelateBrowserPreRequest: RequestMiddleware = async function () {
```

Comment:
1:1 move to have this middleware run sooner

Reply:
do you have to move it to make it run sooner? Couldn't you just move it up in the export list?